test/controllers/pensions/lifetimeAllowance/AnnualAndLifetimeAllowanceCYAViewHelperTest.scala

```diff
@@ -0,0 +1,394 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.pensions.lifetimeAllowance
+
+import models.pension.charges.{LifetimeAllowance, PensionAnnualAllowancesViewModel, PensionLifetimeAllowancesViewModel, UnauthorisedPaymentsViewModel}
+import org.scalatest.Assertion
+import org.scalatest.matchers.should._
+import org.scalatest.wordspec.AnyWordSpec
+import play.api.i18n.Lang
+import play.i18n.{Langs, Messages}
+import play.test.Helpers.stubMessagesApi
+import uk.gov.hmrc.govukfrontend.views.viewmodels.content.HtmlContent
+import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.{ActionItem, Key, SummaryListRow, Value}
+
+class AnnualAndLifetimeAllowanceCYAViewHelperTest extends AnyWordSpec with Matchers {
+
+  val taxYear = 2022
+
+  implicit val messages: Messages = stubbedMessages()
+  "Getting the summary rows" should {
+    "return the expected" when {
+      "you have selected yes for 'reduced annual allowance question'" in {
+        val annualModel = PensionAnnualAllowancesViewModel(
+          reducedAnnualAllowanceQuestion = Some(true),
+          moneyPurchaseAnnualAllowance = None,
+          taperedAnnualAllowance = None,
+          aboveAnnualAllowanceQuestion = None,
+          aboveAnnualAllowance = None,
+          pensionProvidePaidAnnualAllowanceQuestion = None,
+          taxPaidByPensionProvider = None,
+          pensionSchemeTaxReferences = None
+        )
+        val lifetimeModel = PensionLifetimeAllowancesViewModel(
+          aboveLifetimeAllowanceQuestion = Some(true)
+        )
+
+        val summaryListRows = AnnualAndLifetimeAllowanceCYAViewHelper.summaryListRows(annualModel, lifetimeModel, taxYear)
+
+        summaryListRows.length shouldBe 7
```

Comment:
I was a bit unsure about this because the rows i'd be asserting are technically incorrect. And are only visible because the question page for "annual lifetime" hasn't been implemented yet

Reply:
Even if (temporarily), it would make that more transparent, and would make the "fix" easier.

Best to add it in.

Reply:
Confirmed.